minimal.js

```diff
@@ -144,7 +150,25 @@ module.exports = function (dirname, keys, opts) {
   }
 
   db.addBoxer = function addBoxer (boxer) {
-    boxers.push(boxer)
+    switch (typeof boxer) {
```

Comment:
`<style>` with function which have multiple signatures I think I prefer one main path like : 

```js
function addBoxer (boxer) {
  if (typeof boxer === 'function') return addBoxer({ value: boxer })
  // this makes it clear we're putting someone back on the *main* path, and reduces the nesting in the fn

  // ... the main path
}
```

Reply:
I pushed this change because I have itchy fingers

Reply:
Sweet, this method is just a copy and paste of the `addUnboxer()` method. Since `addUnboxer()` used the `switch...case` pattern I used the same here, but I think the important bit is that they're consistent. 🤷

Reply:
yeah I saw that when I started editing. I tried changing `addUnboxer` to drop the switch but it's not possible there. The problem is that addUnboxer accepts 2 radically different modes - a onestep unbox or a two-step unbox. sigh